core/src/main/kotlin/DokkaBootstrapImpl.kt

```diff
@@ -7,8 +7,8 @@ import java.util.function.BiConsumer
 
 
 fun parsePerPackageOptions(args: List<String>): List<PackageOptions> = args.map { it.split(",") }.map {
```

Comment:
I think this check is now unnecessary as we dropped global settings and I see no reason why the regex could not be an `all match`